### PR TITLE
Replace or remove references to Launchpad where possible.

### PR DESCRIPTION
**Pull request by gunchleoc**

### CODE REVIEW

**Review by Yeppisunatra — APPROVED** [review]

**Review by Yeppisunatra — APPROVED** [review]

**Review by Yeppisunatra — APPROVED** [review]

File: debian/copyright [hunk 1]
```diff
@@ -209,9 +209,9 @@ License: GPL-2+
  The media files of Widelands are distributed under the same license
  than the game itself. The blender source files are not included in the same
  source tree because of their size but can be retrieved online from
- https://bugs.launchpad.net/widelands-media/
+ https://launchpad.net/widelands-media/
```
Comment (klaus-halfmann) [inline debian/copyright:212]: Uhm, did we not transfer this to github?

**Review by klaus-halfmann — COMMENTED** [review]
All changes are fine, will checkout the source.

One question inline

File: debian/copyright (hunk 1 above)
Comment (gunchleoc) [reply debian/copyright:212]: No. We left it on Launchpad for now because of the big file sizes.

**Comment by klaus-halfmann** [comment]
The following files still refer to launchpad, what shall happen to them?

CREDITS
ChangeLog
debian/changelog
debian/watch
debian/copyright
po/mp_scenario_island_hopping.wmf/... multiple
po/map_the_green_plateau.wmf/... multiple
po/mp_scenario_smugglers.wmf/... multiple
src/network/gamehost.cc
src/logic/map_objects/bob.cc
src/logic/map_objects/tribes/militarysite.cc
src/logic/map_objects/tribes/worker.cc
src/graphic/render_queue.cc
src/third_party/minizip/ioapi.h
src/sound/sound_handler.cc
utils/create_cppcheck_report
xdg/org.widelands.Widelands.appdata.xml

**Comment by gunchleoc** [comment]
It's either issues that were closed and thus not migrated or stuff that I don't know how to fix right now.

**Review by klaus-halfmann — APPROVED** [review]
So lets have it

**Comment by klaus-halfmann** [comment]
For one plattform test/maps/lua_persistence.wmf/scripting/test_persistence.lua ...  fails ?

**Comment by gunchleoc** [comment]
That failure is https://github.com/widelands/widelands/issues/3585, nothing to do with this branch. Thanks for the review!

**Pull request merged**